webapp/src/shared/context/GamePairingContext.tsx

```diff
@@ -30,15 +34,36 @@ export interface GamePairingContextType {
 export const GamePairingContext = createContext<GamePairingContextType>(null!);
 
 export const GamePairingProvider = ({ children }: { children: ReactNode }) => {
+  const { pathname } = useLocation();
+  const isInQueuePage =
+    pathname.slice(0, PLAY_GAME_QUEUE.length) === PLAY_GAME_QUEUE;
+  const isInGamePage =
+    pathname.slice(0, PLAY_GAME_URL.length + 1) === `${PLAY_GAME_URL}/`;
   const { authUser } = useAuth();
-  const { navigate } = useNavigation();
+  const { goBack, navigate } = useNavigation();
   const { warn, notify } = useNotificationContext();
   const [gameCtx, setGameCtx] = useState<GamePairingContextType>({
     isWaitingToPlay: false,
     isPlaying: false,
     gameRoomId: null,
   });
 
+  useEffect(() => {
+    if (gameCtx.isPlaying && !isInGamePage) {
+      navigate(`${PLAY_GAME_URL}/${gameCtx.gameRoomId}`);
```

Comment:
This is unstable. If users are paired, user game status context and or navigation flickers.

Reply:
This is obsolete and removed. So i'm closing it.